### PR TITLE
Material ui registration formik - Accept this before material-ui-login-formik

### PR DESCRIPTION
**Pull request by dionys93**
# Description

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [] This change requires a documentation update

## Change Status

- [ ] Complete, tested, ready to review and merge
- [X] Complete, but not tested (may need new tests)
- [X] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?

- [ ] Test A
- [ ] Test B

# Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] There are no merge conflicts

### CODE REVIEW

File: mission-control-client/package.json [hunk 1]
```diff
@@ -3,21 +3,21 @@
...
-    "react-scripts": "3.2.0",
-    "validator": "^11.1.0",
+    "react-redux": "^7.1.1",
     "react-router-dom": "^5.1.2",
     "react-scripts": "3.2.0",
     "redux": "^4.0.4",
     "redux-thunk": "^2.3.0",
     "semantic-ui-css": "^2.4.1",
     "semantic-ui-react": "^0.88.1",
-    "validator": "^11.1.0"
```
Comment (agordon1123) [inline mission-control-client/package.json:21]: Do we need the validator package?

**Review by danbergelt — APPROVED** [review]

**Review by agordon1123 — APPROVED** [review]

File: mission-control-client/src/components/auth/Login.js [hunk 2]
```diff
@@ -1,100 +1,70 @@
...
+    errors,
+    touched,
+    values,
+    handleSubmit,
+    status
+}) {
+    const history = useHistory();
+    return (
+        <Form
+            history={history}
```
Comment (danbergelt) [inline mission-control-client/src/components/auth/Login.js:18]: @zoverlvx what's the purpose of this prop?

**Review by danbergelt — APPROVED** [review]

File: mission-control-client/package.json (hunk 1 above)
Comment (dionys93) [reply mission-control-client/package.json:21]: No, that was something old that I was using. Yup is taking over for validator.

File: mission-control-client/src/components/auth/Login.js (hunk 2 above)
Comment (dionys93) [reply mission-control-client/src/components/auth/Login.js:18]: I'm taking the history hook from react-router and passing it into the form so I can then use it in the handleSubmit. At least this is how it should be working in theory.

**Review by armandoroman1016 — APPROVED** [review]

**Pull request merged**